CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Improve documentation and code quality for RISC-V assembly instructions
```

Comment:
Please, remove this file and add a brief description of your changes in `riscv/CHANGELOG.md`